content/CORE/UIReference/Storage/Pools/DatasetsScreen.md

```diff
@@ -129,18 +130,8 @@ Use **ADD ACL ITEM** to add another set of the ACL permission settings.
 
 Select **Apply permissions recursively** to apply the ACL settings recursively to all directories and files in the current dataset.
 
-**USE PERMISSIONS EDITOR** to return to the [**EDIT Permissions**](#dataset-edit-permissions-screen) screen.
-
-## Additional Information
-
-[Pools Screens]({{< relref "/CORE/UIReference/Storage/Pools/PoolsScreens.md" >}})
-
-[Creating Pools]({{< relref "/CORE/CORETutorials/Storage/Pools/PoolCreate.md" >}})
-
-[Managing Pools]({{< relref "/CORE/CORETutorials/Storage/Pools/ManagingPools.md" >}})
-
-[Importing Pools]({{< relref "/CORE/CORETutorials/Storage/Pools/PoolImport.md" >}})
-
-[Setting Up Permissions]({{< relref "/CORE/CORETutorials/Storage/Pools/Permissions.md" >}}) 
+**USE PERMISSIONS EDITOR** to return to the [**EDIT Permissions**](#dataset-edit-permissions-screen) screen. 
```

Comment:
I had to check to see if this was a UI element as the sentence was awkward from that perspective, suggest this as an alternate:
```suggestion
**USE PERMISSIONS EDITOR** opens the [**EDIT Permissions**](#dataset-edit-permissions-screen) screen. 
```